### PR TITLE
Adding timezone

### PR DESCRIPTION
**Pull request by oblandford-novetta**
Adding option for timezone as well as reading command line arguments

### CODE REVIEW

File: ChrootCfg.sh [hunk 1]
```diff
@@ -15,23 +15,138 @@
...
+      printf '\t%-20s%s\n' '--timezone' 'See "-z" short-option'
+   )
+   exit "${SCRIPTEXIT}"
+}
+
+######################
+## Main program-flow
+######################
+OPTIONBUFR=$( getopt \
+   -o Ff:hm:z: \
```
Comment (lorengordon) [inline ChrootCfg.sh]: I don't see these being handled: `-f`, `-F`, `--fstype`, `--no-fips`. What's the plan there?

Comment (oblandford-novetta) [reply ChrootCfg.sh]: I forgot to remove them.

Comment (oblandford-novetta) [reply ChrootCfg.sh]: updated

File: ChrootCfg.sh [hunk 2]
```diff
@@ -15,23 +15,138 @@
...
+
+# Print out a basic usage message
+function UsageMsg {
+   local SCRIPTEXIT
+   SCRIPTEXIT="${1:-1}"
+
+   (
+      echo "Usage: ${0} [GNU long option] [option] ..."
+      echo "  Options:"
+      printf '\t%-4s%s\n' '-f' 'Filesystem-type of chroot-devs (e.g., "xfs")'
```
Comment (lorengordon) [inline ChrootCfg.sh]: Let's remove them from the help message also then...

**Review by lorengordon — COMMENTED** [review]
@oblandford-novetta this largely looks fine, but can you kick of a spel-ci build pointing at this branch, just to be sure?

**Comment by oblandford-novetta** [comment]
> @oblandford-novetta this largely looks fine, but can you kick of a spel-ci build pointing at this branch, just to be sure?

Ran the build pointing to my repo. Build finished successfully.

**Comment by lorengordon** [comment]
@oblandford-novetta It looks like that run was executing the el8 jobs, but this work is on amigen7....


<html>
<body>


SPEL_BUILDERS | minimal-rhel-8-hvm,minimal-centos-8-hvm
-- | --



</body>
</html>

**Comment by lorengordon** [comment]
Oh, I see it now, you ran both.

**Review by lorengordon — APPROVED** [review]
🚀

**Pull request merged**